.phpcs.xml.dist

```diff
@@ -10,6 +10,7 @@
 	<!-- Exclude the code in the PHPCS 2.x test files copied in from PHPCS. -->
 	<exclude-pattern>/Test/AllTests.php</exclude-pattern>
 	<exclude-pattern>/Test/Standards/*.php</exclude-pattern>
+	<exclude-pattern>/bin/class-ruleset-test.php</exclude-pattern>
```

Comment:
`bin/` is typically for shell scripts or other binaries.

Reply:
I'd be happy to place it somewhere else, any suggestions on a more appropriate location ?

The `/Test` directory came to mind, though all the other files in that directory will be removed in WPCS 2.x when PHPCS 2.x is dropped and the test files would need another name if placed there so as not to confuse PHPUnit.